hieradata/class/jenkins.yaml

```diff
@@ -1,5 +1,7 @@
 ---
 
+govuk_jenkins::version: '2.19.2'
```

Comment:
I wonder if it's worth getting ahead of the curve and trying an upgrade to the latest?